### ISSUE TITLE
Problem with current location

### ISSUE THREAD

**Issue by pagyx93** [EddyVerbruggen/nativescript-directions#14]
Hi, I've a problem when I'm using the default "from" (current location).
The problem is when maps app start, it doesn't automatically recognize my current location generating an error.
There's a screenshot of what I'm trying to explain. Thanks

![photo_2017-10-27_15-48-14](https://user-images.githubusercontent.com/15138149/32107229-468b58b2-bb2e-11e7-8806-bbf01a27bc88.jpg)

**Comment by EddyVerbruggen** [comment]
Ehm, that's a problem in Google Maps then..

**Comment by mosid** [comment]
Hi @EddyVerbruggen
I had the same problem, when running on other languages it breaks.( Tried it on android only)
but i managed to get it working by setting the source variable to null
in directions.common.js file in line 7
and changed the following
```javascript
            qs += encodeURIComponent(source)
```
To this
```javascript
        if(source){
            qs += encodeURIComponent(source)
        };
```
But i don't know if it works on IOS
What do you think?
I am willing to make a pull request if you  wish

**Comment by lordrepha1980** [comment]
Your solution works in IOS too. Thanks

**Comment by EddyVerbruggen** [comment]
Thanks for sharing you experiences, and @mosid, thanks a LOT for sharing that solution 💯 

Fix available in 1.1.2 from npm.
